packages/ui/src/utilities/formatRichTextLexical.ts

```diff
@@ -0,0 +1,30 @@
+type SerializedLexicalEditor = {
+  root: {
+    children: Array<{ children?: Array<{ type: string }>; type: string }>
+  }
+}
+
+export function isSerializedLexicalEditor(value: unknown): value is SerializedLexicalEditor {
+  return typeof value === 'object' && 'root' in value
+}
+
+export function formatRichTextLexical(
```

Comment:
I'd change the naming of this function to something more descriptive. And out-of-context "format rich text" doesn't really tell me what it does.

Maybe we can have `utilities/formatDocTitle/index.ts` and `utilities/formatDocTitle/formatLexicalDocTitle.ts`?

That way the location of this utility function will clarify what it's for.